ProcessList.h

```diff
@@ -63,6 +63,7 @@ typedef struct ProcessList_ {
 
    unsigned long long int totalMem;
    unsigned long long int usedMem;
+   unsigned long long int availableMem;
```

Comment:
Platform-dependent fields normally should also go into the platform-dependent struct (in this case `LinuxProcessList`).